lib/utils/__tests__/narrowFixRange.test.mjs

```diff
@@ -306,6 +306,71 @@ describe('narrowFixRange', () => {
 				'a {--foo: abcabc}',
 			);
 		});
+		test('declaration with trailing semicolon', () => {
+			const node = postcss.parse('a { opacity: 45%; }');
+
+			const fixData = {
+				range: [4, 17],
+				text: 'opacity: 0.45;',
```

Comment:
```suggestion
				text: node.toString(),
```

We can use `node.toString()` instead of a hard coded value to better mimic the implementation in `report()`

Reply:
Well `opacity: 0.45;` is not `node.toString()`, it's `opacity: 45%`? But `fixData.text` should be the node _after_ the rule applied the fix, so I'll leave it as it is, let me know if I'm missing something.

Reply:
Sorry I should have elaborated :)

```js
const node = postcss.parse('a { opacity: 45%; }').first.first;

node.value = '0.45';

const fixData = {
	range: [4, 17],
	text: node.toString(),
};
```

There are three changes here:
- using the declaration as the node
- using the PostCSS API's to change the value to `0.45`
- using `node.toString()` to get the updated text

`node.toString()` will result in `opacity: 0.45` (without semicolon).
But `narrowFixRange` is expected to fix that by amending the `text`.

Reply:
Got it! Thanks :)